src/Options.php

```diff
@@ -686,6 +686,24 @@ public function setTracesSampler(?callable $sampler): void
         $this->options = $this->resolver->resolve($options);
     }
 
+    /**
+     * Gets the amount of memory to reserve in case of a fatal error.
+     * The number is in bytes. Null indicated the default should be used.
+     */
+    public function getFatalReservedMemory(): ?int
```

Comment:
I suggest to rename this method to `getFatalErrorHandlerReservedMemoryAmount()`. Also, why not returning `ErrorHandler::DEFAULT_RESERVED_MEMORY_SIZE` instead of null to indicate the default value?

Reply:
Fair point. Change incoming.